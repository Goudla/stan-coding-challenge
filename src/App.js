import React, { Component } from 'react';
import { BrowserRouter } from 'react-router-dom';
import { Route } from 'react-router-dom';

// Scenes
import Root from './scenes/Root';

// Styles
import './App.css';

class App extends Component<*> {
  render() {
    return (
      <BrowserRouter>
        <Route path="/" component={Root} />
      </BrowserRouter>
    );
  }
}

export default App;
