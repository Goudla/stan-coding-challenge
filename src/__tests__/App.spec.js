import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

// Scenes
import App from '../App';

describe('<App />', () => {
  it('Should render', () => {
    const renderer = new ShallowRenderer();
    renderer.render(<App />);
    const result = renderer.getRenderOutput();
    expect(result).toMatchSnapshot();
  });
});
