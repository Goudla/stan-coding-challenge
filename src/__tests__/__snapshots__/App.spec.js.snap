// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> Should render 1`] = `
<BrowserRouter>
  <Route
    component={[Function]}
    path="/"
  />
</BrowserRouter>
`;
